docs/rules/object-shorthand.md

```diff
@@ -98,6 +98,37 @@ You can set the option in configuration like this:
 }
 ```
 
+While set to `"always"`, `"methods"`, or `"properties"`, shorthand syntax using string literal keys can be ignored (for both properties and methods) using the optional parameter `"avoidQuotes"`. Note: The first parameter must be specified when using this optional parameter.
+
+```json
+{
+    "object-shorthand": ["error", "always", { "avoidQuotes": true }]
+}
+```
+
+The following will warn when `"avoidQuotes"` is enabled:
```

Comment:
We are using different terminology here now. See http://eslint.org/docs/rules/comma-dangle#never